velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -335,11 +371,109 @@ TEST_F(ParquetTpchTest, q6) {
   const auto& duckDBRounded =
       fmt::format("select round(revenue, 2) from ({})", duckDbSql);
 
-  auto task = assertQuery(params, filePath, sourcePlanNodeId, duckDBRounded);
+  auto task =
+      assertQuery(params, {filePath}, {sourcePlanNodeId}, duckDBRounded);
 
   const auto& stats = task->taskStats();
   // There should be two pipelines
   ASSERT_EQ(2, stats.pipelineStats.size());
   // We used the default of 10 splits
   ASSERT_EQ(10, stats.numFinishedSplits);
 }
+
+TEST_F(ParquetTpchTest, q18) {
+  const auto& lineitemFilePath = parquetFilesPaths_["lineitem"];
+  const auto& ordersFilePath = parquetFilesPaths_["orders"];
+  const auto& customerFilePath = parquetFilesPaths_["customer"];
+
+  TableColumnsMap tableCols;
+  tableCols["lineitem"] = {"l_orderkey", "l_quantity"};
+  tableCols["orders"] = {
+      "o_orderkey", "o_custkey", "o_orderdate", "o_totalprice"};
+  tableCols["customer"] = {"c_name", "c_custkey"};
+
+  auto lineitemRowType = getRowType("lineitem", tableCols, lineitemFilePath);
+  auto ordersRowType = getRowType("orders", tableCols, ordersFilePath);
+  auto customerRowType = getRowType("customer", tableCols, customerFilePath);
+
+  CursorParameters params;
+  params.maxDrivers = 4;
+  params.numResultDrivers = 1;
+
+  auto planNodeIdGenerator = std::make_shared<PlanNodeIdGenerator>();
+  core::PlanNodeId cSrcNodeId, oSrcNodeId, lSrcNodeId;
+
+  auto big_orders = PlanBuilder(planNodeIdGenerator)
```

Comment:
`bigOrders` (Use camelCase for local variables) https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md

Reply:
What is preferred way to run this script https://github.com/facebookincubator/velox/blob/main/scripts/run-clang-tidy.py ? Does this warn of such above conventions ?

Reply:
just regular invocation with file as argument seems to have worked for now, although it gave 68K roughly, don't think they are just about this change.

Any case, I changed the variable name.